### PR TITLE
Allow for sending credentials on an insecure channel if explicitly configured to do so

### PR DESCRIPTION
**Pull request by josephschorr**

### CODE REVIEW

File: src/util.ts [hunk 1]
```diff
@@ -1,14 +1,90 @@
...
+        this.channelCredentials._equals(other.channelCredentials) &&
+        this.callCredentials._equals(other.callCredentials)
+      );
+    } else {
+      return false;
+    }
+  }
+}
+
+// Create our own known insecure channel creds.
```
Comment (samkim) [inline src/util.ts:44]: Add a note that this is for testing only

File: src/util.ts [hunk 2]
```diff
@@ -1,14 +1,90 @@
 import * as grpc from "@grpc/grpc-js";
+import { ConnectionOptions } from 'tls';
+
+
+// NOTE: Copied from channel-credentials.ts in gRPC Node package because its not exported.
+class ComposedChannelCredentialsImpl extends grpc.ChannelCredentials {
```
Comment (samkim) [inline src/util.ts:6]: Add link far the specific revision of channel-credentials.ts

**Review by samkim — APPROVED** [review]
LGTM, just a couple requests for additional comments

File: src/util.ts (hunk 1 above)
Comment (josephschorr) [reply src/util.ts:44]: Well, some people might not use it for testing if they are within a VPC

**Pull request merged**